### PR TITLE
[Netflix] - Update + Optimization

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/NetflixQualityCheck/NetflixCheckMaxCps.cs [hunk 1]
```diff
@@ -9,20 +10,20 @@ public class NetflixCheckMaxCps : INetflixQualityChecker
...
-                    var charactersPerSeconds = Utilities.GetCharactersPerSecond(p);
+                    int textLenExludingNonDisplayChars = GetTextLengthExcludingNonDisplayChars(p.Text);
+                    double charactersPerSeconds = textLenExludingNonDisplayChars / p.Duration.TotalSeconds;
                     if (charactersPerSeconds > controller.CharactersPerSecond)
                     {
                         var fixedParagraph = new Paragraph(p, false);
-                        while (Utilities.GetCharactersPerSecond(fixedParagraph) > controller.CharactersPerSecond)
-                        {
-                            fixedParagraph.EndTime.TotalMilliseconds++;
-                        }
```
Comment (ivandrofly) [inline libse/NetflixQualityCheck/NetflixCheckMaxCps.cs]: not really fan of this code 😕 ... the newer will do all the work on a constant time

File: libse/NetflixQualityCheck/NetflixCheckMaxCps.cs [hunk 2]
```diff
@@ -33,5 +34,38 @@ public void Check(Subtitle subtitle, NetflixQualityController controller)
             }
         }
 
+
+        public static int GetTextLengthExcludingNonDisplayChars(string text)
```
Comment (ivandrofly) [inline libse/NetflixQualityCheck/NetflixCheckMaxCps.cs]: O(n)

**Comment by niksedk** [comment]
It's not really done... will be changed to something like "Fix common errors" at some point.

File: libse/NetflixQualityCheck/NetflixCheckMaxCps.cs (hunk 1 above)
Comment (niksedk) [inline libse/NetflixQualityCheck/NetflixCheckMaxCps.cs]: Yeah, should be possible to calculate...

**Pull request closed without merging**